src/specialOffer/SpecialOffer.story.mdx

```diff
@@ -0,0 +1,101 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+
+import { SpecialOffer } from './index'
+
+<Meta title="SpecialOffer" />
+
+# **SpecialOffer**
```

Comment:
Below are three possible APIs for the component: 

1) Compound version
In my mind this is the version that offer the better dev experience and the right amount of flexibility for the future (no fear of having a enormous list of props to maintain in 6 months to handle special conditions). But this is also the one that offer the less security (can the dev add multiple Picture ? No LinkCTA at all ?)

2) full props version
This is the full opposite of 1). Very secured from a Typescript point of view. We can even type the precise amount of links / infos we allow.

3) Mix compound / props version
I tried to propose a "take all advantages of the two others options" option but it's maybe "all the disadvantages" 🙃 


What do you think? I find it hard to choose between the options.
Also don't hesitate to propose better names!

Reply:
I vote for 2. Compound APIs are falsely generic (but much nicer looking) and you need to write a bunch of code to assert the non-explicit contract (or to manage all the cases if you allow every combinations which seems a bad idea :) ).

Reply:
Yep. I agree with the problem you just exposed and I think I would vote for option 2 too for such a small component

Reply:
I'm kinda lost since I have the very same issue on the `SEA Landing Page Grid`, as we discussed this morning.
What I understand of these components that **are not PIXAR** is that they don't need flexibility because they aren't meant to be reused in _other_ cases. cf this PR https://github.com/blablacar/ui-library/pull/1056
So, I may **vote for 2**. I'll keep Compound Way just for styled-components elements and for the exposure of the component itself, typescript. Sounds like the better of the two world too hehe

Reply:
Ok let's go for option 2 ! 🚀